package.json

```diff
@@ -128,10 +128,10 @@
     "@types/uuid": "^3.4.0",
     "@types/winston": "^2.2.0",
     "@types/xml2js": "^0.4.0",
-    "electron": "1.7.11",
-    "electron-builder": "19.48.3",
+    "electron": "1.8.2",
+    "electron-builder": "19.56.0",
```

Comment:
Be mindful that we only use this for the Linux installer creation (which isn't done by CI currently).

Reply:
ok, I will revert this back to the previous version to see if it plays nice with Electron 1.8.